prior/__init__.py

```diff
@@ -134,9 +135,41 @@ def _get_git_lfs_cmd():
         return git_lfs_path
 
 
-def _clone_repo(
+def _get_git_auth_token() -> Optional[str]:
+    if gh_auth_token is not None:
+        # Treats gh_auth_token as a string
+        return gh_auth_token.strip()  # type: ignore
+    elif os.environ.get("GITHUB_TOKEN") is not None:
+        return os.environ["GITHUB_TOKEN"].strip()
+    else:
+        # look at ~/.git-credentials
+        git_credentials_path = f"{os.environ['HOME']}/.git-credentials"
+        token = None
+        if os.path.exists(git_credentials_path):
+            with open(git_credentials_path, "r") as f:
+                tokens = f.read()
+
+            try:
+                token = next(token for token in tokens.split("\n") if token.endswith("github.com"))
+                token = token.split(":")[2]
+                token = token.split("@")[0]
+            except StopIteration:
+                logging.warning(
+                    "No github token found, attempting to use unauthenticated request to GitHub."
+                    " If you're using a private repo or have exceeded the API limit, please"
+                    " add your GitHub token as an environment variable (`export GITHUB_TOKEN=YOUR_TOKEN`),"
```

Comment:
This sometimes doesn't work for me. I think we should consider removing this option and just having people pass in `gh_auth_token=YOUR_TOKEN`.